packages/extension-koni-ui/src/types/account.ts

```diff
@@ -22,6 +22,7 @@ export enum AccountSignMode {
   QR = 'qr',
   LEGACY_LEDGER = 'legacy-ledger',
   GENERIC_LEDGER = 'generic-ledger',
+  ECDSA_SUBSTRATE_LEDGER = 'ecdsa-substrate-ledger',
```

Comment:
Remove AccountSignMode in extension-koni-ui, then use AccountSignMode from extension-base